src/query/settings/src/settings_default.rs

```diff
@@ -692,7 +692,14 @@ impl DefaultSettings {
                 }),
                 ("parse_datetime_ignore_remainder", DefaultSettingValue {
                     value: UserSettingValue::UInt64(1),
-                    desc: "Ignore trailing chars when parse string to datetime(disable by default)",
+                    desc: "Ignore trailing chars when parse string to datetime",
+                    mode: SettingMode::Both,
+                    range: Some(SettingRange::Numeric(0..=1)),
+                }),
+                //parse_datetime_ignore_remainder
+                ("force_timestamp_conversion", DefaultSettingValue {
```

Comment:
some idea about the name of the setting,  ignore me if wrong:

1.  The name of the setting is too generic. Could you choose a more specific name?
2.  not only for type `time_stamp`? 
3.  I see another setting `parse_datetime_ignore_remainder` , maybe they can share the same naming pattern? maybe we will have other time related settings later.

Reply:
rename `force_timestamp_conversion` -> `enable_dst_hour_fix` is better? cc @sundy-li